### PR TITLE
Add a dev success stories section

### LINKED ISSUE #1386: Implement Developer stories section
**Is your feature request related to a problem? Please describe.**
- As discussed in the recent standup calls and hive, we need to implement the developer stories section on SEF site. The implementation will be similar to ScholarX success stories section.

**Describe the solution you'd like**
- Implement developer stories section using the following mockup design. (please refer to scholarX success stories section as well).

MacBook Air - 1

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by RandilaP**
## Purpose

The purpose of this PR is to fix #1386

## Goals
- Add a section to show engineering team success stories

## Approach
- Build a template using JS 
- Gather information using google sheet

### Screenshots
image

  
### Preview Link

https://pr-1494-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by RandilaP** [comment]
@jayasanka-sack @anjula-sack Can you review my PR.

File: assets/content/static/navbar.html [hunk 1]
```diff
@@ -70,7 +70,7 @@ <h6 class="text-primary mb-md-1">Handbook</h6>
                                         <h6 class="text-primary mb-md-1">ScholarX</h6>
                                     </div>
                                 </a>
-                                <a href="/engineering-team.html" class="media d-flex align-items-center">
+                                <a href="../../../engineering-team/index.html" class="media d-flex align-items-center">
```
Comment (YoshithaRathnayake) [inline assets/content/static/navbar.html]: We don't need the `/index.html` part right? @RandilaP @anjula-sack 
```suggestion
                                <a href="../../../engineering-team" class="media d-flex align-items-center">
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: engineering-team/index.html [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+        const content = Mustache.render(template, {card: data});
+        document.getElementById('stories').innerHTML = content;
+    }
+
+    loadStories();
+</script>
+<script>
+    $(function () {
+        loadNavAndFooter('../assets/content/static');
+        // loadBlogPosts();
```
Comment (anjula-sack) [inline engineering-team/index.html]: Why this is commented?

File: engineering-team/index.html [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+            </div>
+        </div>
+    </div>
+    {{/card}}
+</script>
+<script>
+    async function loadStories() {
+        const response = await fetch("https://script.google.com/macros/s/AKfycbwv81hbfTsHAGcNQnmrjcV9YPVkipHRbjXct_6DlPWrovCVqy90j8DlRNb2D_MTQAiOuQ/exec");
+        const data = await response.json();
+        const template = document.getElementById("template-dev").innerHTML;
```
Comment (anjula-sack) [inline engineering-team/index.html]:
```suggestion
        const template = document.getElementById("success-story").innerHTML;
```

File: engineering-team/index.html [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+        const content = Mustache.render(template, {card: data});
+        document.getElementById('stories').innerHTML = content;
+    }
+
+    loadStories();
+</script>
+<script>
```
Comment (anjula-sack) [inline engineering-team/index.html]:
```suggestion
```

File: engineering-team/index.html [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+    async function loadStories() {
+        const response = await fetch("https://script.google.com/macros/s/AKfycbwv81hbfTsHAGcNQnmrjcV9YPVkipHRbjXct_6DlPWrovCVqy90j8DlRNb2D_MTQAiOuQ/exec");
+        const data = await response.json();
+        const template = document.getElementById("template-dev").innerHTML;
+        const content = Mustache.render(template, {card: data});
+        document.getElementById('stories').innerHTML = content;
+    }
+
+    loadStories();
```
Comment (anjula-sack) [inline engineering-team/index.html]: Move this inside line 265

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: engineering-team/index.html (hunk 2 above)
Comment (RandilaP) [reply engineering-team/index.html]: It was already commented

File: engineering-team/index.html (hunk 5 above)
Comment (RandilaP) [reply engineering-team/index.html]: okay

File: assets/content/static/navbar.html (hunk 1 above)
Comment (RandilaP) [reply assets/content/static/navbar.html]: yeah. I'll remove that.

File: engineering-team/index.html (hunk 2 above)
Comment (anjula-sack) [reply engineering-team/index.html]: Remove it then

Comment (anjula-sack) [reply engineering-team/index.html]: Remove the empty `engineering-team/styles.css` file as well

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: engineering-team/index.html (hunk 2 above)
Comment (RandilaP) [reply engineering-team/index.html]: Okay

File: assets/content/static/navbar.html (hunk 1 above)
Comment (YoshithaRathnayake) [inline assets/content/static/navbar.html]: Again we don't need this `index.html` part no? @RandilaP @anjula-sack 
```suggestion
                                <a href="../../../engineering-team/" class="media d-flex align-items-center">
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by RandilaP** [comment]
@YoshithaRathnayake Can u review this again and point out what I missed.

**Comment by YoshithaRathnayake** [comment]
> @YoshithaRathnayake Can u review this again and point out what I missed.

Sure @RandilaP

**Review by YoshithaRathnayake — COMMENTED** [review]
Looks good for me @RandilaP 
If you have finished doing all changes, you can make this PR open except draft

@anjula-sack any changes requested?

**Comment by RandilaP** [comment]
Okay Thank you @YoshithaRathnayake

**Review by anjula-sack — APPROVED** [review]
Good job! @RandilaP  As we discussed yesterday let's keep this PR open till we get the stories. 

cc: @YoshithaRathnayake @Piumal1999

**Review by YoshithaRathnayake — DISMISSED** [review]
Nice work for me too
Thank You! @RandilaP

**Comment by anjula-sack** [comment]
@RandilaP can you resolve the conflicts?

File: engineering-team/index.html [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+        <div id="posts" class="posts"></div>
+        <div class="row justify-content-center" id="btnPosts" style="visibility:hidden;">
+            <button id="posts-show-more" class="btn btn-link btn-outline-white btn-info">SEE MORE</button>
+        </div>
+    </div>
+</section>
+<!--developer success stories section-->
+<section class="dev-success">
+    <div class="container">
+        <h1 class="display-3 text-center my-2">Engineering Team Success Stories</h1>
```
Comment (anjula-sack) [inline engineering-team/index.html]:
```suggestion
        <h1 class="display-3 text-center my-2">Success Stories</h1>
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: engineering-team/index.html [hunk 7]
```diff
@@ -0,0 +1,273 @@
...
+        {{/data}}
+    </div>
+</script>
+<script id="success-story" type="text/html">
+    {{#card}}
+    <div class="project-card shadow my-4 ">
+        <div class="container">
+            <div class="row  py-5">
+                <div class="col-md-3 align-items-center justify-content-center text-center my-auto">
+                    <img src="{{image}}}"
```
Comment (anjula-sack) [inline engineering-team/index.html]:
```suggestion
                    <img src="{{image}}"
```

File: engineering-team/index.html [hunk 8]
```diff
@@ -0,0 +1,273 @@
...
+                         style="width: 100px; height: 100px; border-radius: 50%; border: 5px solid #EAECEE;">
+                    <a href="{{linkedin}}}" target="_blank"><p class="pt-2 font pr-2">{{dev_name}}</p></a>
```
Comment (anjula-sack) [inline engineering-team/index.html]:
```suggestion
                    <a href="{{linkedin}}" target="_blank"><p class="pt-2 font pr-2">{{dev_name}}</p></a>
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: engineering-team/index.html (hunk 8 above)
Comment (RandilaP) [reply engineering-team/index.html]: @anjula-sack Done

Comment (anjula-sack) [reply engineering-team/index.html]: No it's not @RandilaP

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by RandilaP** [comment]
@RandilaP Sorry. There was an error

File: engineering-team/index.html [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+    <!-- global CSS -->
+    <link type="text/css" href="/assets/css/global-custom.css" rel="stylesheet">
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+
+=======
+>>>>>>> e4e7140 (Add requested changes)
+=======
+>>>>>>> origin/devsuccess
```
Comment (anjula-sack) [inline engineering-team/index.html]:
```suggestion
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by Piumal1999** [comment]
> Good job! @RandilaP As we discussed yesterday let's keep this PR open till we get the stories.
> 
> cc: @YoshithaRathnayake @Piumal1999

I also added a success story. @RandilaP could you please resolve the conflicts, so we can merge this Pull request?

**Review by anjula-sack — APPROVED** [review]
Thank you! @RandilaP

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Can you resolve the conflicts @RandilaP?

**Comment by RandilaP** [comment]
@anjula-sack Conflict resolved

File: assets/content/static/navbar.html [hunk 10]
```diff
@@ -70,7 +72,7 @@ <h6 class="text-primary mb-md-1">Handbook</h6>
                                         <h6 class="text-primary mb-md-1">ScholarX</h6>
                                     </div>
                                 </a>
-                                <a href="/engineering-team.html" class="media d-flex align-items-center">
+                                <a href="../../../engineering-team/" class="media d-flex align-items-center">
```
Comment (Piumal1999) [inline assets/content/static/navbar.html]: This one also would work
```suggestion
                                <a href="/engineering-team" class="media d-flex align-items-center">
```

**Review by Piumal1999 — DISMISSED** [review]
Hi, @RandilaP I requested a minor change.

**Review by anjula-sack — APPROVED** [review]
Thank you! @RandilaP

**Pull request merged**